File: tebangka.java
import java.util.Scanner;
import java.util.Random;

public class TebakAngka {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Random random = new Random();
        
        int angkaRahasia = random.nextInt(100) + 1;
        int tebakan = 0;
        
        while (tebakan != angkaRahasia) {
            System.out.print("Tebak angka antara 1 dan 100: ");
            tebakan = scanner.nextInt();
            if (tebakan < angkaRahasia) {
                System.out.println("Tebakan terlalu rendah!");
            } else if (tebakan > angkaRahasia) {
                System.out.println("Tebakan terlalu tinggi!");
            }
        }
        System.out.println("Selamat! Tebakan kamu benar.");
    }
}

File: tebangka.rb
def tebak_angka
  angka_rahasia = rand(1..100)
  tebakan = nil

  while tebakan != angka_rahasia
    print "Tebak angka antara 1 dan 100: "
    tebakan = gets.to_i
    if tebakan < angka_rahasia
      puts "Tebakan terlalu rendah!"
    elsif tebakan > angka_rahasia
      puts "Tebakan terlalu tinggi!"
    end
  end

  puts "Selamat! Tebakan kamu benar."
end

tebak_angka
